.github/workflows/main.yml

```
@@ -29,15 +37,28 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
 
-    - name: Install standard dependencies
+    - name: Install Operational dependencies
+      if: ${{ matrix.test_config == 'Ops'}}
       run: |
-        pip install -r test_requirements.txt
+        pip install numpy==${{ matrix.numpy_ver }}
         pip install -r requirements.txt
+        pip install -r test_requirements.txt
```

```suggestion
        pip install "cdflib<1.0"
        pip install -r requirements.txt
        pip install -r test_requirements.txt
```
I think this will fix the operational test flow. cdflib 1.0+ uses type hinting, which numpy 1.19 doesn't support. Looking back through the various jobs where only this task fails, I saw this warning pop up. This is how I'm handling the operational flow at pysatNASA.